service-descriptions/petstore-openrpc.json

```diff
@@ -120,7 +120,7 @@
          "name": "pets",
         "description": "Expected response to a valid request",
         "schema": {
-          "$ref": "#/components/schemas/Pets"
+          "$ref": "#/components/schemas/Pet"
```

Comment:
Could you please 
1. fix the formatting
2. fix the name from pets to pet

Thank you :)